registry/etcdv3/listener.go

```diff
@@ -0,0 +1,88 @@
+package etcdv3
+
+import (
+	"context"
+	"strings"
+)
+
+import (
+	"github.com/apache/dubbo-go/common"
+	"github.com/apache/dubbo-go/common/logger"
+	"github.com/apache/dubbo-go/registry"
+	"github.com/apache/dubbo-go/remoting"
+)
+
+import (
+	"github.com/juju/errors"
```

Comment:
don't use juju/errors

Reply:
Thx, i had change github.comjuju/errors to github.com/pkg/errors